server/mediaserver.go

```diff
@@ -105,21 +105,27 @@ type authWebhookResponse struct {
 
 func NewLivepeerServer(rtmpAddr string, httpAddr string, lpNode *core.LivepeerNode) *LivepeerServer {
 	opts := lpmscore.LPMSOpts{
-		RtmpAddr: rtmpAddr, RtmpDisabled: true,
-		HttpAddr: httpAddr,
-		WorkDir:  lpNode.WorkDir,
+		RtmpAddr:     rtmpAddr,
+		RtmpDisabled: true,
+		HttpAddr:     httpAddr,
```

Comment:
We can probably omit `HttpAddr` here (and from the `LivepeerServer` as well), since the `httpAddr` value from CLI should still be in scope when `http.ListenAndServe` is called

Reply:
Interesting. So how would we access `httpAddr` here? 

```golang
ec <- http.ListenAndServe(s.httpAddr, s.HTTPMux)
```

Reply:
Does `ec <- http.ListenAndServe(*httpAddr, s.HTTPMux)` work?

Reply:
`Mediaserver.go` doesn't seem to have access to `*httpAddr`. It's undefined

Reply:
Probably @josh mean to change `StartMediaServer` to be `StartMediaServer(ctx context.Context, transcodingOptions, httpAddre string)`.
Anyway `httpAddr` is not used in `NewLivepeerServer` directly.

Reply:
OH yes, we could do that @darkdarkdragon !